### PR TITLE
Updated content

### PR DESCRIPTION
**Pull request by ronny-mysten**
## Description 

Updating content

## Test Plan 

Locally

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: sdk/docs/pages/typescript-sdk/cryptography/keypairs.mdx [hunk 1]
```diff
@@ -1,109 +1,114 @@
...
-import { ED25519KeyPair, ED25519PublicKey } from '@mysten/sui.js/keypairs/ed25519';
+// random Keypair
+const keypair = new Ed25519Keypair();
+// Keypair from an existing secret key (Uint8Array)
+const keypair = Ed25519Keypair.fromSecretKey(secretKey);
+```
 
-const keyPair = new ED25519KeyPair();
-const bytes = keyPair.getPublicKey().toBytes();
+With your key pair created, you can use it to perform actions on the network. For example, you can use it to sign transactions, like the following code that creates and signs a personal message using the public key from the key pair created in the previous code:
```
Comment (hayes-mysten) [inline sdk/docs/pages/typescript-sdk/cryptography/keypairs.mdx:31]: Keypairs will never act on the network.  They are purely running cryptography operations locally (in the future we may have keypairs that act as a proxy run the operations in a wallet or worker, but there won't be any interaction with the sui network

File: sdk/docs/pages/typescript-sdk/cryptography/multisig.mdx [hunk 2]
```diff
@@ -1,26 +1,20 @@
-import { Callout } from 'nextra/components';
+# Multi-Signature Transactions
 
-# MultiSig
+The Sui TypeScript SDK provides a `MultiSigPublicKey` class to support [Multi-Signature](https://docs.sui.io/learn/cryptography/sui-multisig) (MultiSig) transaction signing.
```
Comment (hayes-mysten) [inline sdk/docs/pages/typescript-sdk/cryptography/multisig.mdx:3]: technically we also support signing personal messages, but this is probably fine as is

File: sdk/docs/pages/typescript-sdk/faucet.mdx [hunk 3]
```diff
@@ -1,21 +1,23 @@
...
+
+Devnet, Testnet, and local networks include faucets that mint SUI. You can use the Sui TypeScript SDK to call a network's faucet and provide SUI to the address you provide.
 
-You can request sui from the faucet using the `@mysten/sui.js/faucet` package.
+To request SUI from a faucet, import the `requestSuiFromFaucetV0` function from the `@mysten/sui.js/faucet` package to your project. 
 
 ```typescript
 import { requestSuiFromFaucetV0, getFaucetHost } from '@mysten/sui.js/faucet';
+```
+Use `requestSuiFromFaucetV0` in your TypeScript code to request SUI from the network's faucet. The `getFaucetHost` function is useful for development, but you should use the actual [Mainnet RPC address](../index#network-locations) as the `host` parameter value in production apps.
```
Comment (hayes-mysten) [inline sdk/docs/pages/typescript-sdk/faucet.mdx:12]: there won't be a faucet in mainnet, so this callout only really applies for setting up a SuiClient and not faucet methods

**Review by hayes-mysten — APPROVED** [review]
This looks great, thanks for updating all of this!  Looks like the linter is not happy with some of the formatting, you can run `pnpm lint:fix` in the docs directory and it should fix any issues.  The only thing you may run into is that it may split a line in transaction-building/basics so that a line starts with import which might need to be prefixed with a `\` to avoid thinking its trying to import something

File: sdk/docs/pages/typescript-sdk/faucet.mdx (hunk 3 above)
Comment (ronny-mysten) [reply sdk/docs/pages/typescript-sdk/faucet.mdx:12]: Ha! Duh. I wasn't thinking...

**Comment by ronny-mysten** [comment]
> The only thing you may run into is that it may split a line in transaction-building/basics so that a line starts with import which might need to be prefixed with a `\` to avoid thinking its trying to import something

fyi, a space works, too.

File: sdk/docs/pages/typescript-sdk/cryptography/keypairs.mdx (hunk 1 above)
Comment (ronny-mysten) [reply sdk/docs/pages/typescript-sdk/cryptography/keypairs.mdx]: Poorly worded on my part. I changed it to what I meant to write, let me know if it still sounds wrong.

**Pull request merged**